web/.storybook/preview.js

```diff
@@ -30,3 +31,14 @@ addParameters({
     },
   },
 })
+
+addParameters({
```

Comment:
I just moved the Provider here. Now it's working for docs aswell!

Reply:
Cool, didn't know how to pass documentation page here. Thanks :)